examples/generate-proofs/generateProofs.go

```diff
@@ -30,8 +32,13 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
-	ctx := context.Background()
-	getProofs(ctx, *web3, contractAddr, strings.Split(string(data), "\n"))
+	ctx, cancel := context.WithTimeout(context.Background(), 60*time.Second)
```

Comment:
Since we are refactoring this with a context I think its also worth it to set this timeout as constants instead of magic number in the middle of the code